.github/workflows/tools.yml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   build-macos-latest:
     name: Build tools with macos latest
-    runs-on: macos-latest
+    runs-on: macos-14
```

Comment:
This was already done by GitHub. They switched to it. 
- https://github.blog/changelog/2024-04-01-macos-14-sonoma-is-generally-available-and-the-latest-macos-runner-image

My PRs are currently failing with it, because macos-latest is now pointing to ARM64... :/ Will submit PR shortly.